heartbeat/IPaddr2

```diff
@@ -701,6 +703,23 @@ log_arp_sender() {
     ocf_log $LOGLEVEL "$output"
 }
 
+# wrapper function to manage PID file to run arping in background
+run_with_pidfile() {
+    local cmdline
+    local pid
+    local rc
+
+    cmdline="$@"
+
+    $cmdline &
+    pid=$!
+    echo "$pid" > $SENDARPPIDFILE
+    wait $pid
```

Comment:
I'm a bit confused here. What's the purpose of running the program in the background when immediately we wait for it?